components/core/src/package/install.rs

```diff
@@ -202,6 +202,19 @@ impl PackageInstall {
         }
     }
 
+    /// Determines whether or not this package has a runnable service.
+    pub fn is_runnable(&self) -> bool {
+        // Currently, a runnable package can be determined by checking if a `run` hook exists in
+        // package's hooks directory or directly in the package prefix.
+        if self.installed_path.join("hooks").join("run").is_file() {
+            true
+        } else if self.installed_path.join("run").is_file() {
+            true
+        } else {
+            false
+        }
```

Comment:
Small style nitpick, but maybe we could collapse this all down:
```rust
pub fn is_runnable(&self) -> bool {
  self.installed_path.join("hooks").join("run").is_file() ||
    self.installed_path.join("run").is_file() 
}
```

Reply:
Good call, I avoided this form until I saw how `rustfmt` aligned it, sold now :)